components/infobox/wikis/ageofempires/infobox_series_custom.lua

```diff
@@ -31,15 +33,20 @@ function CustomSeries.run(frame)
 	return series:createInfobox()
 end
 
+---@return WidgetInjector
 function CustomSeries:createWidgetInjector()
 	return CustomInjector()
 end
 
-function CustomSeries.addToLpdb(series, lpdbData)
```

Comment:
`CustomSeries.addToLpdb(self, lpdbData)` is (in this context) the same as `CustomSeries:addToLpdb(lpdbData)`
but since we prefer the `:` and imo annotating a param that is never used and could be avoided ...